src/mgr/DaemonServer.cc

```diff
@@ -505,15 +505,15 @@ bool DaemonServer::handle_report(const ref_t<MMgrReport>& m)
 
 
   {
-    lock.lock();
+    std::unique_lock locker(lock);
```

Comment:
minor nit: better would be `std::scoped_lock`

Reply:
yep